velox/docs/monthly-updates/2021/october-2021.rst

```diff
@@ -17,7 +17,7 @@ Core Library
   * Access query configs.
   * Allow for providing fast path for all-ASCII inputs.
 
-  More details in :doc:`How to add a scalar function? <../develop/scalar-functions>` guide.
+  More details in :doc:`How to add a scalar function? <../../develop/scalar-functions>` guide.
```

Comment:
The Sphinx documentation says, the target name can be relative to the documentation’s root folder
https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role

Can you try if `/develop/scalar-functions` works?
We can then avoid this relative path change in the future.